workers/social/lib/social/models/group/index.coffee

```diff
@@ -385,6 +386,54 @@ module.exports = class JGroup extends Module
         callback null, memberData
 
 
+  # joinUser
+  #
+  # Joins user with given options to group either by logging in or converting
+  # them.
+  #
+  # @return {ResponseWithToken}
+  #
+  @joinUser = ->
+  @joinUser = secure (client, options, callback) ->
```

Comment:
Tests should show that it's working exactly the same as `jointeam` handler, so it's not open to everyone as long as `allowedDomains` property is not set to `['*']`